Sources/TuistCore/Graph/LibraryNode.swift

```diff
@@ -62,7 +62,7 @@ public class LibraryNode: PrecompiledNode {
         return libraryNode
     }
 
-    public override var binaryPath: AbsolutePath {
+    public override func binaryPath() throws -> AbsolutePath {
```

Comment:
I don't see why `throws` is required here 🤔

Reply:
context can be found at https://github.com/tuist/tuist/pull/701#discussion_r352316534